src/main/java/org/jboss/modules/Main.java

```diff
@@ -76,6 +75,7 @@ public final class Main {
     }
 
     private static final String[] NO_STRINGS = new String[0];
+    private static final int JVM_VERSION = Runtime.version().feature();
```

Comment:
I would reuse the `actualVersion` variable on line ~366 (move it up a little if needed) and drop the constant field.